### PR TITLE
feat: Make the role work on Debian based systems

### PR DESCRIPTION
**Pull request by Thulium-Drake**
Enhancement: Make role work on Debian

Reason: I have a few systems that run Debian and I wanted to configure Postfix on them in the same manner as I do RHEL based systems.

Result: Role now works on Debian (note that SELinux and Firewalld are not supported/tested)

Issue Tracker Tickets (Jira or BZ if any):

### CODE REVIEW

**Comment by spetrosi** [comment]
[citest]

File: tasks/main.yml [hunk 1]
```diff
@@ -111,6 +111,8 @@
         cp /etc/postfix/main.cf
         /etc/postfix/main.cf.{{ postfix_backup_multiple |
         ternary("$(date -Iseconds)", "backup") }}
+      args:
+        executable: /bin/bash
```
Comment (spetrosi) [inline tasks/main.yml:115]: Why is this needed? Does Debian use some other executable?

Comment (spetrosi) [reply tasks/main.yml:115]: Actually, this task copies files, and we can use the `copy` module instead of using `shell`. There is an old #2 that nobody looked at for years, but it has some good code that would fix this. Can you use it here?

Comment (spetrosi) [reply tasks/main.yml:115]: I am about to merge #2, which will fix the issue with pipefail. This PR now lacks adding compatibility with Debian in meta/main.yml, like in https://github.com/linux-system-roles/ssh/blob/main/meta/main.yml#L18-L23

Comment (Thulium-Drake) [reply tasks/main.yml:115]: Cool, that will also fix it :)

Shall I close this one then?

**Comment by richm** [comment]
Will fix by https://github.com/linux-system-roles/postfix/pull/2 instead
